### DISCUSSION TITLE
Error after upload file on Upload page with ESP32

### DISCUSSION THREAD

**Discussion by nikanet** [Hieromon/AutoConnect#407] [Q&A]
I use ESP32 and LittleFS. After upload file like .css or images on Upload AC page all files uploaded by Platformio before no longer available via http (path like /file.css) and uploaded file also not available. On ESP8266 everything is fine but on ESP32 have this issue. 

Also with sketch https://github.com/Hieromon/AutoConnect/tree/master/examples/FileUpload not able to acces file by uri /somefile.some on ESP32 after upload.

Please help.

**Comment by Hieromon** [comment]
@nikanet At the time you posted this issue, the ESP32 LittleFS was integrated into core 2.0.0. AutoConnect has supported ESP32 LittleFS since v130. And I released it after your post.
Which branch did you use?
Did you correctly declare the definition macro to [make AutoConnect use the ESP32 LittleFS](https://hieromon.github.io/AutoConnect/filesystem.html#selecting-appropriate-filesystem)?
Also, FileUpload example is hard-coded to use SPIFFS for ESP32. Did you point out this after understanding the contents of the sketch?

**Comment by Hieromon** [comment]
This topic may be an issue based on the wrong composition, so it will be moved to the discussion once. It will be returned to ISSUE when the AutoConnect issue becomes apparent.
